arm-documentdb/2015-04-08/swagger/documentdb.json

```diff
@@ -0,0 +1,776 @@
+          "type": "string",
+          "description": "The type of Azure resource."
+        },
+        "location": {
+          "type": "string",
+          "description": "The location of the resource group to which the resource belongs."
+        },
+        "tags": {
+          "$ref": "#/definitions/Tags"
+        }
+      },
+      "required": [
+        "location"
+      ],
+      "x-ms-azure-resource": true
+    },
+    "DatabaseAccount": {
+      "description": "A DocumentDB database account.",
+      "type": "object",
+      "properties": {
+        "kind": {
+          "description": "Indicates the type of database account. This can only be set at database account creation.",
+          "type": "string",
+          "default": "GlobalDocumentDB",
+          "enum": [
+            "GlobalDocumentDB",
+            "MongoDB",
+            "Parse"
+          ],
+          "x-ms-enum": {
+            "name": "DatabaseAccountKind",
+            "modelAsString": true
+          }
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/DatabaseAccountProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "#/definitions/Resource"
+        }
+      ]
+    },
+    "ConsistencyPolicy": {
+      "type": "object",
+      "description": "The consistency policy for the DocumentDB database account.",
+      "properties": {
+        "defaultConsistencyLevel": {
+          "description": "The default consistency level and configuration settings of the DocumentDB account.",
+          "type": "string",
+          "enum": [
+            "Eventual",
+            "Session",
+            "BoundedStaleness",
+            "Strong"
+          ],
+          "x-ms-enum": {
+            "name": "DefaultConsistencyLevel",
+            "modelAsString": false
+          }
+        },
+        "maxStalenessPrefix": {
+          "description": "When used with the Bounded Staleness consistency level, this value represents the number of stale requests tolerated. Accepted range for this value is 1 – 2,147,483,647. Required when defaultConsistencyPolicy is set to 'BoundedStaleness'.",
+          "type": "integer",
+          "minimum": 1,
+          "maximum": 2147483647,
+          "format": "int64"
+        },
+        "maxIntervalInSeconds": {
+          "description": "When used with the Bounded Staleness consistency level, this value represents the time amount of staleness (in seconds) tolerated. Accepted range for this value is 1 - 100. Required when defaultConsistencyPolicy is set to 'BoundedStaleness'.",
+          "type": "integer",
+          "minimum": 1,
+          "maximum": 100,
+          "format": "int32"
+        }
+      },
+      "required": [
+        "defaultConsistencyLevel"
+      ]
+    },
+    "DatabaseAccountProperties": {
+      "description": "Properties for the database account.",
+      "type": "object",
+      "properties": {
+        "provisioningState": {
```

Comment:
We don't have an enum for provisioningState over here. Whereas we have it for "Location.provisioningState". 
I would recommend not having an enum  anywhere in the spec specially for "provisioningState" as it does more harm than any good (w.r.t the property being readOnly (the iser cannot set it) and the values can potentially change if the service decides to add/remove/modify the states).